plugin/hover.py

```diff
@@ -248,6 +233,26 @@ def symbol_actions_content(self, listener: AbstractViewListener, point: int) ->
         actions = [lk.link(point, self.view) for lk in link_kinds if self.provider_exists(listener, lk)]
         return " | ".join(actions) if actions else ""
 
+    def link_content_and_range(self) -> Tuple[str, Optional[sublime.Region]]:
+        if len(self._document_links) > 1:
+            combined_region = range_to_region(self._document_links[0]["range"], self.view)
+            for link in self._document_links[1:]:
+                combined_region = combined_region.cover(range_to_region(link["range"], self.view))
+            if all(link.get("target") for link in self._document_links):
+                return '<a href="quick-panel:DocumentLink">Follow Link…</a>', combined_region
+            else:
+                return "Follow Link…", combined_region
+        elif len(self._document_links) == 1:
+            link = self._document_links[0]
+            target = link.get("target")
+            label = "Follow Link" if link.get("target", "file:").startswith("file:") else "Open in Browser"
```

Comment:
I would probably go with `Open in Browser` if starts with `http` and otherwise `Follow Link` because there could be cases where link would use a custom protocol like `virtual:` or something and it would be a link to a local content.

Reply:
Hmm, but I see that `_on_navigate` handler would then have to be somehow changes to follow that logic...